atom/common/platform_util.h

```diff
@@ -23,7 +23,7 @@ void OpenItem(const base::FilePath& full_path);
 
 // Open the given external protocol URL in the desktop's default manner.
 // (For example, mailto: URLs in the default mail user agent.)
-bool OpenExternal(const GURL& url);
+bool OpenExternal(const GURL& url, const bool without_activation);
```

Comment:
The `const` is not needed here, because `bool` is passed by value not reference.